ldif.go

```diff
@@ -0,0 +1,256 @@
+package ldap
+
+import (
+	"bufio"
+	"bytes"
+	"encoding/base64"
+	"errors"
+	"fmt"
+	"io"
+	// "os"
+	"strconv"
+)
+
+// A basic LDIF parser. This one does currently just support LDIFs like
+// they're generated by tools like ldapsearch(1) / slapcat(8). Change
+// records are not supported.
+type LDIF struct {
+	RelaxedParser bool // ignore parsing errors in a line
+	Entries       []*Entry
+	curLine       int
```

Comment:
use of `curLine`,`version`, and `changeType` seems local to Parse... do they need to be member fields?